T8Suite/Form1.cs

```diff
@@ -8396,6 +8435,44 @@ public SearchReplaceTuningPackage(string searchReplace)
                 foundS1 = inputStr.IndexOf('{') + 1;
                 foundS2 = inputStr.IndexOf('}');
                 string [] replaceString = inputStr.Substring(foundS1, foundS2 - foundS1).Split(',');
```

Comment:
Search and Replace code looks rather similar. Maybe its possible to extract a method that both use?

Reply:
this is part of the search and replace code, not sure what you  mean :)

Reply:
Done